src/scope/core/pipelines/streamdiffusionv2/modular_blocks.py

```diff
@@ -35,7 +37,29 @@
     ]
 )
 
+# Text mode: pure text-to-video, no video-dependent blocks. We still run
+# EmbeddingBlendingBlock so that conditioning_embeds is produced for the
+# DenoiseBlock, matching the behaviour of the video path and other pipelines.
+TEXT_BLOCKS = InsertableDict(
```

Comment:
Just leaving a quick comment before actually reviewing design:

Any particular reason why we can't use AutoPipelineBlocks from modular diffusers to have a single set of blocks and just conditionally switch on presence of a video input to determine whether to use/include V2V specific blocks eg PrepareVideoLatentsBlock, PreprocessVideo, etc.?

I'm asking about this because in my mind if possible it seems simpler to be able to to have a single set of blocks with conditional including/exclusion of certain blocks depending on the workflow being used which is determined by whether there is a video input (which I believe AutoPipelineBlocks is supposed to handle).